fabric-chaincode-shim/src/main/java/org/hyperledger/fabric/Logging.java

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+package org.hyperledger.fabric;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Logger class to use throughout the Contract Implementation
+ *
+ */
+public class Logging {
```

Comment:
We intentionally removed any logging implementation from Go chaincode ... why do we think we need to provide one when people can pick their own logging package?   I know there was already one in here, but really I think we should remove it.  If we feel the need to log internally, then we should just have a private logger